### DISCUSSION TITLE
Open source is fashion（开源作为时尚品牌）

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#50] [5️⃣ 全球化]
转一个David 老师提供的 Ted 视频：[Lessons from fashion's free culture](https://www.ted.com/talks/johanna_blakley_lessons_from_fashion_s_free_culture?language=en)

Copyright law's grip on film, music and software barely touches the fashion industry ... and fashion benefits in both innovation and sales, says Johanna Blakley. In her talk, she talks about what all creative industries can learn from fashion's free culture.

她是把 Fashion 当做科技产品在做，四处告人抄袭。Fashion 是不能有版权的产业，开源是 fashion 的基本模式。这个 TED 不错，谈 fashion 是如何在开源下繁荣的。从传统品牌到 H&M/Zara 到 Shein ~

开源可以向 fashion 学习很多，

![image](https://user-images.githubusercontent.com/15010826/170413031-1bc19532-33a4-4ddb-9449-3ed51feb3d06.png)

再推荐一个老电影，讲的就是 fashion 设计如何开源传播，新的想法被创出来，然后透过一层一层的拷贝和改造，到最后成为大众接受的。有钱的买 LV 的，没钱的从淘宝淘，每个阶层都可以用上。LV 不抱怨淘宝货，淘宝货也不卖 LV 价格。最终大品牌卖的是那个 logo，淘宝卖的才是美丽。

现在的耳机就是这样，每个人都能负担起 TWS 耳机，可以装 X 买 Apple，可以 80 元买华强北货。现在开源的问题就是 LV 抱怨淘宝货，说淘宝货抢了他们的客户。都在想如何大公司的高价服务也要，单独用户的小钱也要。全海里的大鱼小鱼都要捞到他的一个网里面。

![5a6384fcc767a1a0cd526aae29f5b7f](https://user-images.githubusercontent.com/15010826/170413512-bb449961-e647-4cc8-9234-42821c478401.jpg)

**Comment by will-ww** [comment]
有个联想，开源社区的一个很大的价值展示都就是品牌与logo，这个东西如果能和时尚结合起来，配合 NFT 这样的技术，是否可以做开源项目的品牌价值度量与资产化呢~
- 开源项目的品牌度量（科学性与共识性）
- 品牌 logo + 行为数据 + 生成艺术
- NFT 市场

![image](https://user-images.githubusercontent.com/15010826/172030220-8bcaf10e-4fa1-4bd1-b4b4-7b2e2faa33e2.png)

![image](https://user-images.githubusercontent.com/15010826/172030268-36d5fa98-cd03-42ca-8198-bea66576440d.png)

**Comment by xgdyp** [comment]
一些有趣的事实：
- github的章鱼猫（octocat）有很多很丰富的表情包https://octodex.github.com
- misbrand这个开源项目做过很多“错版”图标https://github.com/mkrl/misbrands
- 据说，twitter的鸟和github的octocat原型都买自iStock，出自同一个设计师之手（而且据说当时设计师只拿到几美元的报酬）
